docs/contributing.md

```diff
@@ -82,3 +102,8 @@ something that is confusing or not covered at all, feel free to
 [rd]: https://docs.radicle.xyz
 [rt]: https://github.com/radicle-dev/radicle-upstream/issues/1958
 [up]: http://docs.radicle.xyz/docs/using-radicle/creating-patches
+[ds]: https://discord.gg/HRdnwAwGbG
+[dn]: https://github.com/radicle-dev/radicle-upstream/issues?q=is%3Aopen+is%3Aissue+label%3Adesign-needed
+[ff]: https://www.figma.com/file/tVqxYmi0YM4TRRebCXWZaF/Radicle-Funding?node-id=393%3A8210
```

Comment:
This opens Figma on your playground. Is that correct?
<img width="1552" alt="Screenshot 2021-09-23 at 12 53 06" src="https://user-images.githubusercontent.com/158411/134495271-fc7590db-a4ef-4ce3-9531-b6aae14411a3.png">

Reply:
nope fixed

Reply:
For some reason it still opens it for me on your playground. 🤷